clang/lib/Sema/SemaDecl.cpp

```diff
@@ -16753,6 +16753,9 @@ ExprResult Sema::VerifyBitField(SourceLocation FieldLoc,
         << FieldName << FieldTy << BitWidth->getSourceRange();
     return Diag(FieldLoc, diag::err_not_integral_type_anon_bitfield)
       << FieldTy << BitWidth->getSourceRange();
+  } else if (FieldTy->isIntCapType()) {
+    return Diag(FieldLoc, diag::err_capability_type_bitfield)
+           << FieldName << FieldTy << BitWidth->getSourceRange();
```

Comment:
FieldName is null for anonymous bitfields; see err_not_integral_type_bitfield vs err_not_integral_type_anon_bitfield